scipy-notebook/Dockerfile

```diff
@@ -14,9 +14,13 @@ RUN apt-get update && \
 
 USER jovyan
 
+# Fix the ipywidgets since the base image has an older version of ipywidgets.
+# The older version works with the Python2 instance but not the Python3 instance.
+# Installing a newer version allows the widgets to work with both Python versions.
+RUN conda install -c conda-forge ipywidgets=5.1.5
```

Comment:
So, I think we should start using a much newer version of `conda`. It appears to be 3.19.1 [currently]( https://github.com/jupyter/docker-stacks/blob/2485724a08c9ce55c296fdd486abdc5244eabe1c/base-notebook/Dockerfile#L63 ), but to get channel priority to work we need 4.1.x. Though this may come with some other surprises. That all being said, I pulled down the `base-notebook` and found it has 4.1.4 installed. So, this pinning is not actually respected. Something we should think about. Though I would recommend using something on the 4.1.x series and we should discuss fixing that pinning in a new PR.

In other words, `conda-forge` should be preferred without this channel argument.